ee/modules/600-flant-integration/hooks/license_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 Flant JSC
+Copyright 2022 Flant JSC
```

Comment:
We don't need to change it. This is the date of creation of this file. [Example](https://github.com/kubernetes/kubernetes/blob/aae07c6f7bb715750a2d0cc77cf81c4e31a4904b/pkg/apis/core/register.go#L2) from Kubernetes repo.